### DISCUSSION TITLE
mess

### DISCUSSION THREAD

**Discussion by hiyasharma01018** [mukul-lokash/1#2] [Q&A] [answered]
why is this  file missed up, how  to make it originised

**Comment by MxkulSharma** [comment, marked as answer]
its simple make folder for every languages you used and keep the main index.html file out then address the files with there new loaction i.e if you used javascript and css make 2 seprate folder named javascript and css and put there specific files in that and chnage the location  of those  files in index.html as javascript/<your file name> and css/<your file name>
